src/components/SiteList.js

```diff
@@ -0,0 +1,44 @@
+import React, { Component } from 'react';
+import summary from '../apis/summary';
+
+import SiteItem from './SiteItem';
+
+class SiteList extends Component {
+  state = {
+    sites: []
+  };
+
+  componentDidMount() {
+    summary.get('/sites').then(response => {
+      this.setState({ sites: response.data.features });
```

Comment:
You don't need to use promises for simple items like this anymore :)

```js
async componentDidMount() {
    const {
      data: { features: sites }
    } = await summary.get('/sites');
    this.setState({ sites });
  }
```

Reply:
this is a bit advance, very neat use of es6 syntax with async/await. Don't you mind explaining this?

Reply:
Oh hey! 

This is the combination of using `await` with `object-destructuring`, followed by `implicit object values`. Kind of like this:

```js
  const response = await summary.get('/sites')
  const { data : { features: sites } } = response
  this.setState({ sites })
```

The first thing is just an await function. You know that. 
The second thing is that I'm pulling `features` from `response.data.features` and renaming it to `sites`
Finally, because the name of the key in "state" is `sites`, and I've declared sites to what I want the thing to be, it means I can use just sites as key and value by mentioning it once.